docs/theme/mlflow/static/css/theme.css

```diff
@@ -5192,7 +5192,7 @@ a .rst-content tt,
 a .rst-content tt,
 a .rst-content code {
     font-weight: bold;
-    color: #052247
+    color: #2980B9;
```

Comment:
@mateiz 

Before:
![image](https://user-images.githubusercontent.com/1400247/45256200-e7ee3c80-b347-11e8-8328-0de4ed6d9fb1.png)

After:
![image](https://user-images.githubusercontent.com/1400247/45256199-e3c21f00-b347-11e8-9a0e-92916a995d80.png)

Seems reasonable to me.